main/index.ts

```diff
@@ -34,26 +35,41 @@ process.env.BUNDLE_LOCATION = process.env.BUNDLE_LOCATION || path.resolve(__dirn
 log.transports.console.level = process.env.LOG_LEVEL || 'info'
 log.transports.file.level = ['development', 'test'].includes(process.env.NODE_ENV) ? false : 'verbose'
 
-function getCrashReportFields () {
+function getCrashReportFields() {
   const fields = ['networks', 'networksMeta', 'tokens']
 
   return fields.reduce((extra, field) => {
     return { ...extra, [field]: JSON.stringify(store('main', field) || {}) }
   }, {})
 }
 
+function getSentryException(event: Event) {
+  const frames = event?.exception?.values?.[0].stacktrace?.frames || []
```

Comment:
should probably loop through all of the exceptions in `values` here instead of only taking the first one